infrastructure/lambda/template.yaml

```diff
@@ -1151,22 +653,6 @@ Resources:
       Value: !FindInMap [PassportCriAudienceMapping, Environment, !Ref 'Environment']
       Description: The passport credential issuer (audience) identifier
 
-  AuthRequestKmsEncryptionKeyIdParameter:
-    Type: AWS::SSM::Parameter
-    Properties:
-      Name: !Sub "/${AWS::StackName}/AuthRequestKmsEncryptionKeyId"
-      Type: String
-      Value: !ImportValue core-infrastructure-CriDecryptionKey1Id
-      Description: The (KMS) encryption key identifier for decrypting authorisation requests
-
-  VerifiableCredentialIssuerParameter:
-    Type: AWS::SSM::Parameter
-    Properties:
-      Name: !Sub "/${AWS::StackName}/verifiable-credential/issuer"
-      Type: String
-      Value: !FindInMap [VerifiableCredentialIssuerMapping, Environment, !Ref 'Environment']
-      Description: Issuer of the Verifiable Credential
-
```

Comment:
The alarm topic will need updated so were not alerting in the real passport channels. Should just need a new unique value her Fn::Sub: '{{resolve:ssm:/alerting/pagerduty-passport/url}}'